pubspec.yaml

```diff
@@ -81,3 +81,4 @@ flutter:
   assets:
     - assets/images/
     - assets/icons/
+    - assets/fonts/
```

Comment:
similar comment before 😛  you may want to use flutter_gen to do all the import for you instead of manually declare in your static class later 👌  your choice.